src/niftyone/analysis_levels/participant.py

```diff
@@ -140,6 +141,7 @@ def _participant_single(
     logging.info(f"Processing subject {sub}")
 
     runner.table = index.filter("sub", sub)
+    mpl.use("agg")
```

Comment:
Similar to another comment, I think the individual figure generation modules should handle their own mpl setup, if they even use mpl.